DESCRIPTION

```diff
@@ -0,0 +1,11 @@
+Package: r-geomstats

```

Comment:
This file shouldn't be here, as it here already in geomstats:master. You should 'rebase' on master.

If not already, start using github command lines, it will be easier for later.
https://gist.github.com/brakmic/d8ee499960082ea2b03e13e7581dd14e